specs/swagger.yml

```diff
@@ -0,0 +1,1397 @@
+        '400':
+          $ref: '#/responses/badRequestResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+  '/terminators/{id}':
+    parameters:
+      - $ref: '#/parameters/id'
+    get:
+      summary: Retrieves a single terminator
+      description: Retrieves a single terminator by id. Requires admin access.
+      tags:
+        - Terminator
+      operationId: detailTerminator
+      responses:
+        '200':
+          $ref: '#/responses/detailTerminator'
+        '404':
+          $ref: '#/responses/notFoundResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+    put:
+      summary: Update all fields on a terminator
+      description: Update all fields on a terminator by id. Requires admin access.
+      tags:
+        - Terminator
+      operationId: updateTerminator
+      parameters:
+        - name: terminator
+          in: body
+          required: true
+          description: A terminator update object
+          schema:
+            $ref: '#/definitions/terminatorUpdate'
+      responses:
+        '200':
+          $ref: '#/responses/updateResponse'
+        '400':
+          $ref: '#/responses/badRequestResponse'
+        '404':
+          $ref: '#/responses/notFoundResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+    patch:
+      summary: Update the supplied fields on a terminator
+      description: Update the supplied fields on a terminator. Requires admin access.
+      tags:
+        - Terminator
+      operationId: patchTerminator
+      parameters:
+        - name: terminator
+          in: body
+          required: true
+          description: A terminator patch object
+          schema:
+            $ref: '#/definitions/terminatorPatch'
+      responses:
+        '200':
+          $ref: '#/responses/patchResponse'
+        '400':
+          $ref: '#/responses/badRequestResponse'
+        '404':
+          $ref: '#/responses/notFoundResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+    delete:
+      summary: Delete a terminator
+      description: Delete a terminator by id. Requires admin access.
+      tags:
+        - Terminator
+      operationId: deleteTerminator
+      responses:
+        '200':
+          $ref: '#/responses/deleteResponse'
+        '400':
+          $ref: '#/responses/badRequestResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+        '409':
+          $ref: '#/responses/cannotDeleteReferencedResourceResponse'
+
+  ###################################################################
+  # Links
+  ##################################################################
+  '/links':
+    get:
+      summary: List links
+      description: |
+        Retrieves a list of link resources; doesn not supports filtering, sorting, or pagination. Requires admin access.
+      tags:
+        - Link
+      operationId: listLinks
+      #      parameters:
```

Comment:
Documentation says it support filter/sort/pagination - but inputs are commented out

Reply:
Doc does it does NOT support those things (with a misspelling). Fix misspelling and removed comment out inputs